index.template.html

```diff
@@ -3916,6 +3925,191 @@ <h5>Inferencing of Personally Identifiable Information Privacy Risk</h5>
 
   </section>
 
+  <section id="thing-templates" class="normative">
+  <h1>Thing Model</h1>
+      
+  <p class="note">
+    The following section has its origin in [[wot-thing-description]], Annex C. Here Thing Description 
+    Template is renamed to <a>Thing Model</a>, but keeps the same intention.
+    For this version of the specification, <a>Thing Model</a> and its model features (e.g., extendions, obligations, placeholder) are formal 
+    introduced.
+  </p>
+
+    <p>
+    A <em>Thing Model</em> is a special kind of a <a>Thing Description</a> with less restriction as 
```

Comment:
I think calling it a "special kind" of Thing Description is not a good way to put this.   I would say it is "related to" a Thing Description, but it is NOT a Thing Description and so can't be a special kind of one.  Note as well this also implies we should not use the same content type (td+jsonld) but should define a new one (tm+jsonld would be good).

Reply:
...is a template (or class) for Thing Description...

Reply:
done